packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -39,15 +41,15 @@ describe('background sync queue test', () => {
 
   let backgroundSyncQueue;
 
-  beforeEach(async function() {
+  beforeEach(async () => {
```

Comment:
These should be functions. Mocha does some weird binding stuff that simply won't work if there are fat arrow functions.